docs/css/main.css

```diff
@@ -325,6 +325,7 @@ a.anchorjs-link:hover { text-decoration: none; }
   border-bottom-width: 0px;
   margin-top: 0px;
   width: 210px;
```

Comment:
Since we're here, shall we batch `left-menu.width` to 210 px too? That makes the subtitles look truncated. I don't know why we made that longer specifically (https://github.com/apache/spark/pull/25459/files#r521829348).

And, actually we should make the width itself more wider. I would suggest to make it 280px. cc @gatorsmile and @dilipbiswal

Reply:
Hi @HyukjinKwon  As I modified, the horizontal scrollbar can be displayed and used after setting the height to 80%
after:
![image](https://user-images.githubusercontent.com/28332082/98903795-3e8ccd80-24f3-11eb-8ecf-6b0ad334bcb3.png)

Reply:
How does that sound @HyukjinKwon

Reply:
That's fine. Can be separately done. but I think we should still fix https://github.com/apache/spark/pull/30335#discussion_r521831406.

Reply:
Ok，Thanks @HyukjinKwon and @srowen , I will submit the code separately as a fix for the next issues.